components/chart/PriceChart.vue

```diff
@@ -23,6 +23,29 @@
       </NeoDropdownItem>
     </NeoDropdown>
 
+    <NeoDropdown
+      :mobile-modal="false"
+      class="chart-setting-icon min-width-fit-content"
+      position="bottom-left">
+      <template #trigger>
+        <NeoButton no-shadow variant="icon">
+          <NeoIcon
+            icon="gear"
+            pack="fas"
+            size="medium"
+            :color="isDarkMode ? 'white' : 'black'" />
```

Comment:
It looks like no need to add the colour. Or we could try `var(--text-color)`
```suggestion 
/>
```

Reply:
you are right. works just the same without
fixed